components/telemetry-operator/api/v1alpha1/logpipeline_types.go

```diff
@@ -147,7 +147,6 @@ func filterOutCondition(conds []LogPipelineCondition, condType LogPipelineCondit
 //+kubebuilder:resource:scope=Cluster
 //+kubebuilder:subresource:status
 //+kubebuilder:printcolumn:name="Status",type=string,JSONPath=`.status.conditions[-1].type`
-//+kubebuilder:printcolumn:name="Unsupported-Plugins",type=boolean,JSONPath=`.spec.enableUnsupportedPlugins`
```

Comment:
Let's keep the print column and show the value from the new status field

Reply:
done.